lib/contention_classification/client.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'contention_classification/configuration'
+
+module ContentionClassification
+  class Client < Common::Client::Base
+    include Common::Client::Concerns::Monitoring
+    configuration ContentionClassification::Configuration
+
+    STATSD_KEY_PREFIX = 'api.contention_classification'
+
+    def classify_vagov_contentions_expanded(params)
+      with_monitoring do
+        perform(:post, Settings.contention_classification_api.expanded_contention_classification_path,
+                params.to_json.to_s, request_headers)
+      end
+    rescue Faraday::Error => e
+      Rails.logger.error("ContentionClassification::Client Faraday error: #{e.message}")
```

Comment:
Could add a bit more context to the error to help in debugging (if it ever comes to it):
```ruby
Rails.logger.error("ContentionClassification::Client Faraday error on #{Settings.contention_classification_api.expanded_contention_classification_path}: #{e.message}")
```